libraries/SDU/extras/SDUBoot/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -x
+
+ARDUINO=arduino-cli
+SKETCH_NAME="SDUBoot.ino"
+SKETCH="$PWD/$SKETCH_NAME"
+BUILD_PATH="$PWD/build"
+OUTPUT_PATH="../../src/boot"
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	ARDUINO="/Applications/Arduino.app/Contents/MacOS/Arduino"
+fi
+
+buildSDUBootSketch() {
+	BOARD=$1
+	DESTINATION=$2
+
+	$ARDUINO compile -b $BOARD --output-dir="$BUILD_PATH" $SKETCH
```

Comment:
```suggestion
	"$ARDUINO" compile -b $BOARD --output-dir="$BUILD_PATH" "$SKETCH"
```

Insufficient quoting on `$SKETCH` would cause the command to fail if `$PWD` contains problematic characters.

The other quotes added in this suggestion are not strictly required, but it is best practices to always quote shell variables whenever possible.